contrib/post-receive

```diff
@@ -12,9 +12,16 @@ echo "\n==> Activating virtualenv…\n"
 . /opt/jarbas.venv/bin/activate
 
 echo "\n==> Installing NodeJS packages…\n"
+if [ -d node_modules/ ]; then
+    rm -rf node_modules/
+fi
+if [ -d elm-stuff/ ]; then
+    rm -rf elm-stuff/
+fi
 yarn install
 
 echo "\n==> Installing Python packages…\n"
+pip install -U pip
 pip install -r requirements.txt
```

Comment:
wouldn't it be case to add a `-U` on here too? by forcing the update of outdated packages?